.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Release'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to be released.'
+        required: false
+        default: ''
+        type: string
+      base-branch:
+        description: 'The branch where we do this release.'
+        required: false
+        default: ''
+        type: string
+
+permissions: {}
+
+jobs:
+  release:
+    uses: OpenVoxProject/shared-actions/.github/workflows/release.yml@main
```

Comment:
This has not been merged into main yet.  Will be functional when https://github.com/OpenVoxProject/shared-actions/pull/12 is merged.